src/App.tsx

```diff
@@ -23,7 +24,9 @@ const App: React.FC = () => {
               <Route path=':pageId' element={<Week4 />}></Route>
             </Route>
           </Route>
-          <Route path='aditya' element={<Aditya />}></Route>
+          <Route path='/aditya' element={<Aditya />}>
+            <Route path='/aditya/week3' element={<AdityaWeek3 />}></Route>
```

Comment:
don't use your name in the other components rather you can write week3Main or something like that. Only initial component name should be of your name

Reply:
Other will also import their weeks, so many week3 variables will be confusing